plugins/registry.js

```diff
@@ -136,4 +138,20 @@ export default class PluginRegistry {
     registerReducer = (reducer) => {
         reducerRegistry.register('plugins-' + this.id, reducer);
     }
+
+    // Register a handler for custom WebSocket events created by the server-side of your plugin.
```

Comment:
Is there a use case for plugins listening for non-custom events? Or events from other plugins?

Reply:
Hmmm good point. I don't see any harm in it and that could be useful. I'll update it to allow for that